GSServer.py

```diff
@@ -71,18 +71,64 @@ def start_server(args, m=MVelKeepConfig()):
         log.error("Failed to load scenario")
         return
 
-    sync_global = TickSync(rate=sim_config.traffic_rate, realtime=True, block=True, verbose=False, label="EX")
+    sync_global = TickSync(rate=sim_config.traffic_rate, realtime=True, block=True, verbose=False, label="EX", usr_input=sim_config.wait_for_input)
     sync_global.set_timeout(sim_config.timeout)
 
+    #find screen info 
+    screens = screeninfo.get_monitors()
+    primary_screen = None
+    for monitor in screens:
+        if monitor.is_primary:
+            primary_screen = monitor
+            break
+
     if sim_config.wait_for_input:
-        input("Press [ENTER] to start...")
+        if not sim_config.show_dashboard:
+            input("Press [ENTER] to start...")
+        else:
+            #create a small window
+            def on_enter(key):
+                if key == keyboard.Key.enter:
+                    start_window.after(0, start_window.quit())
+            
+            pos_x = primary_screen.x
+            pos_y = primary_screen.y
+            
+            start_window = tk.Tk()
+            set_width = 300
+            set_height = 200
+
+            if args.dash_pos:
+                #place in the middle of the dashboard
+                pos_x = args.dash_pos[0] + args.dash_pos[2] // 2 - set_width // 2
+                pos_y = args.dash_pos[1] + args.dash_pos[3] // 2 - set_height // 2
+            else:
+                pos_x += (primary_screen.width - set_width) // 2
+                pos_y += (primary_screen.height - set_height) // 2
+            
+            # Apply position
+            start_window.geometry(f"{set_width}x{set_height}+{int(pos_x)}+{int(pos_y)}")
+
+            #set window text
+            instructions = tk.Label(start_window, text="Press [ENTER] to start...")
+            instructions.pack(expand=True)
+
+            start_window.lift()
+            start_window.attributes('-topmost', True)
+            start_window.focus_force()
```

Comment:
yes, I like that it's focused automatically and I can just press Enter without having to switch to the window.